src/core/renderLayer.js

```diff
@@ -0,0 +1,25 @@
+
+export default class RenderLayer {
+    constructor() {
+        this.dataframes = [];
```

Comment:
Any special reason to use an array? 
If we use an object we get constant add/delete time instead linear

Reply:
Because "premature optimization is the root of all evil" :slightly_smiling_face: 
If we get to the point where we need O(1) for deleting the reference to a dataframe is that we already have f***ed it up, a lot.